src/masonite/utils/helpers.py

```diff
@@ -539,3 +539,28 @@ def password(password_string):
     return bytes(
         bcrypt.hashpw(bytes(password_string, "utf-8"), bcrypt.gensalt())
     ).decode("utf-8")
+
+
+def optional(attribute, default=None, is_method=False):
+    """Wrap an object on which any attributes/methods can be called
+    without raising an error if it does not exist. It will return a default value of None instead,
+    which can be overriden.
+
+    If you want to call method on the wrapped object, is_method should be set to True."""
+
+    class OptionalWrapper(object):
```

Comment:
Classes in Python 3 don't need to inherit the object class. This can be removed. 

```suggestion
    class OptionalWrapper:
```